README.md

```diff
@@ -485,6 +485,51 @@ deleteAll() async {
 ```
 
 
+## Trouble-Shooting Section
+
+The *first* time I tried to change the path of the Flutter folder so that it could be accessed by the terminal without having to use several paths, it **did not work**, every time I closed the terminal Flutter was no longer recognized.</br> 
+
+![Screen Shot 2020-02-10 at 10 35 56](https://user-images.githubusercontent.com/27420533/74142666-41b7fd80-4bf1-11ea-9916-6b16261349aa.png)
+
+Whenever I used *this* command:</br>
+
+
+![Screen Shot 2020-02-10 at 10 15 19](https://user-images.githubusercontent.com/27420533/74141184-5c3ca780-4bee-11ea-89e8-32cf116d304d.png)
```

Comment:
When you do include a screenshot, don't simply use the name applied by your operating system "_Screen Shot 2020-02-10 at 10 15 19_" that is not very informative to the end user or the person reviewing/maintaining/extending the guide.

Always _rename_ the screenshot file with a descriptive name so that it's clear what the screenshot _is_.
e.g:
```suggestion
![flutter-export-path](https://user-images.githubusercontent.com/27420533/74141184-5c3ca780-4bee-11ea-89e8-32cf116d304d.png)
```

But as noted above, I think this should be a code snippet _instead_ of a screenshot:

```suggestion
`export PATH="$PATH:~/development/flutter/bin`
```
That way the person _reading_ your "Troubleshooting Guide" can copy-paste the solution much faster!

Reply:
I realize now, I thought since it was the command line I just needed to put the screenshot on.

Reply:
Code is almost _always_ better than an image. it loads faster and is easier to maintain. 👍